app/modules/app-service-front-door/front-door-rules.tf

```diff
@@ -0,0 +1,6 @@
+resource "azurerm_cdn_frontdoor_rule_set" "default" {
+  count = var.environment == "dev" ? 1 : 0

```

Comment:
I wonder if we can hoist all of these conditionals out of the module entirely - and put them in back-office-appeals.tf?

So then we don't have to deal with any counts or brackets everywhere.